pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -688,6 +688,16 @@ func getDataSecAtRestInfo(ctx *zedagentContext) *info.DataSecAtRest {
 	return ReportDataSecAtRestInfo
 }
 
+func addConfigItemsToDeviceInfo(deviceInfoPtr *info.ZInfoDevice,
```

Comment:
Not sure I understand.. This is just a function - that is getting called from PublishDeviceInfoToZedCloud. instead of putting all that code in that one big function - I spun that part off into a separate function..  THis is not parsing config. 

Is the name confusing?? Or did I mis-understood u r comment?